eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -11,14 +11,15 @@
 
     <!-- These are what the analyzers are built against. They are overridden in full source build.
          It may be possible to generate SBRPs -->
+    <UsagePattern IdentityGlob="Microsoft.CodeAnalysis.Analyzers/*2.9.4*" />
     <UsagePattern IdentityGlob="Microsoft.CodeAnalysis.AnalyzerUtilities/*3.3.0*" />
-    <UsagePattern IdentityGlob="Microsoft.CodeAnalysis.Analyzers*" />
-    <UsagePattern IdentityGlob="Microsoft.CodeAnalysis.Common*" />
-    <UsagePattern IdentityGlob="Microsoft.CodeAnalysis.CSharp*" />
-    <UsagePattern IdentityGlob="Microsoft.CodeAnalysis.CSharp.Workspaces*" />
-    <UsagePattern IdentityGlob="Microsoft.CodeAnalysis.Workspaces.Common*" />
+    <UsagePattern IdentityGlob="Microsoft.CodeAnalysis.Common/*3.3.1*" />
+    <UsagePattern IdentityGlob="Microsoft.CodeAnalysis.CSharp/*3.3.1*" />
+    <UsagePattern IdentityGlob="Microsoft.CodeAnalysis.CSharp.Workspaces/*3.3.1*" />
+    <UsagePattern IdentityGlob="Microsoft.CodeAnalysis.Workspaces.Common/*3.3.1*" />
```

Comment:
Should these 3.3.1 packages also be in SBRP?

Reply:
According to @mmitche, these are executed at build time so we don't want them in SBRP since those are reference implementation only.

Reply:
> We may end up with packages that should be SBRPs for some repos, and not SBRPs for others.

We have had the case where a reference package satisfies on repo's requirements but not another because it is loaded.  In those cases the repo that loads the assembly would need to use a live/current version.